router/core/errors.go

```diff
@@ -175,48 +175,49 @@ func propagateSubgraphErrors(ctx *resolve.Context) {
 // writeRequestErrors writes the given request errors to the http.ResponseWriter.
 // It accepts a graphqlerrors.RequestErrors object and writes it to the response based on the GraphQL spec.
 func writeRequestErrors(r *http.Request, w http.ResponseWriter, statusCode int, requestErrors graphqlerrors.RequestErrors, requestLogger *zap.Logger) {
-	if requestErrors != nil {
-		wgRequestParams := NewWgRequestParams(r)
-		if wgRequestParams.UseSse {
-			setSubscriptionHeaders(wgRequestParams, r, w)
-
-			if statusCode != 0 {
-				w.WriteHeader(statusCode)
-			}
-			_, err := w.Write([]byte("event: next\ndata: "))
-			if err != nil {
-				if requestLogger != nil {
-					if rErrors.IsBrokenPipe(err) {
-						requestLogger.Warn("Broken pipe, error writing response", zap.Error(err))
-						return
-					}
-					requestLogger.Error("Error writing response", zap.Error(err))
-				}
-				return
-			}
-		} else if wgRequestParams.UseMultipart {
-			// Handle multipart error response
-			if err := writeMultipartError(w, requestErrors, requestLogger); err != nil {
-				if requestLogger != nil {
-					requestLogger.Error("error writing multipart response", zap.Error(err))
-				}
-			}
-			return
-		}
+	if requestErrors == nil {
```

Comment:
I just removed nesting by returning early. Previously, the whole block was wrapped in `err != nil`